core/test-src/org/pentaho/di/compatibility/ValueTest.java

```diff
@@ -764,7 +764,7 @@ public void testGetXML() {
     vs6.setPrecision( 2 );
     result = vs6.getXML();
     assertEquals(
-      "<value><name>Name</name><type>Date</type><text>2006&#x2f;03&#x2f;01 17&#x3a;01&#x3a;02.005</text><length>-1</length><precision>2</precision><isnull>N</isnull></value>",
+      "<value><name>Name</name><type>Date</type><text>2006/03/01 17:01:02.005</text><length>-1</length><precision>2</precision><isnull>N</isnull></value>",
```

Comment:
So clearly, the old ESAPI encoder encoded colon( : ) and forward slash ( / ). I'm a bit worried about backward compatibility here if we have *any* persisted XML as it clearly won't be decoded back into colon and forward slash. Please get to the bottom of what in the whole system might have used this - if it's a method used in serializing a KTR/KJB, we could be creating a regression.